contracts/src/v0.8/functions/dev/1_0_0/interfaces/IFunctionsBilling.sol

```diff
@@ -43,10 +44,14 @@ interface IFunctionsBilling {
     int256 fallbackNativePerUnitLink;
   }
 
-  // @notice Gets the configuration of the Chainlink Functions billing registry
+  // @notice Gets the Chainlink Coordinator's billing configuration
   // @return config
   function getConfig() external view returns (Config memory);
 
+  // @notice Sets the Chainlink Coordinator's billing configuration
+  // @param config - See the contents of the Config struct in IFunctionsBilling.Config for more information
+  function updateConfig(Config memory config) external;
```

Comment:
Similar question about if we want to commit to configuration in the interface long term. I'd argue we don't.